gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthServer.js

```diff
@@ -85,10 +82,29 @@ gpii.oauth2.oauth2orizeServer.listenOauth2orize = function (oauth2orizeServer, c
     }));
 
     oauth2orizeServer.exchange(oauth2orize.exchange.clientCredentials(function (client, scope, done) {
-        var clientCredentialsPromise = authorizationService.grantClientCredentialsAccessToken(client.id, scope);
+        var clientCredentialsPromise = authorizationService.grantPrivilegedPrefsCreatorAuthorization(client.id, scope);
         gpii.oauth2.oauth2orizeServer.promiseToDone(clientCredentialsPromise, done);
     }));
 
+    oauth2orizeServer.exchange(oauth2orize.exchange.password(function (client, username, password, scope, done) {
+        var passwordPromise = authorizationService.grantGpiiAppInstallationAuthorization(username, client.id);
+
+        var authorizationMapper = function (authorization) {
+            return authorization.accessToken;
+        };
+
+        var paramsMapper = function (params) {
+            // extra parameters to be included in the `oauth2orizeServer` response except `accessToken`
+            delete params.accessToken;
```

Comment:
fluid.censorKeys might be preferable here, as not modifying the client's structure